ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -176,19 +160,23 @@ object Assertions {
     val actualRetryabilitySeconds = actualErrorDetails
       .collectFirst { case err: ErrorDetails.RetryInfoDetail => err.retryDelayInSeconds }
 
+    if (!actualErrorId.contains(expectedErrorId))
```

Comment:
Hmm, this was a problem already before, but what happens when expected error code is a substring of the actual?

Reply:
That was initially an `Option`.. When changed to String, it still typechecked. Thanks, will change

Reply:
It makes sense. Thanks